src/extension/Daemon.js

```diff
@@ -346,6 +346,16 @@ var Daemon = class Daemon {
 
     return 0;
   }
+  
+  // Opens a menu with that name if there are currently none open. Closes the currently 
```

Comment:
I think `clang-format` fails due to this trailing space here:

```suggestion
  // Opens a menu with that name if there are currently none open. Closes the currently
```